<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myorders</class>
 <widget class="QMainWindow" name="myorders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запросы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 180,220);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leSearch">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32,120, 215);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAll">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>24</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="two">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>771</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,120, 115);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Вывести книги абонента&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Фамилия абонента&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>teResult</zorder>
   <zorder>btnSelectAll</zorder>
   <zorder>two</zorder>
   <zorder>btnMenu</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>leSearch</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
